shopelectro/tests/tests_views.py

```diff
@@ -516,3 +519,32 @@ def test_admin_autocomplete_has_no_model_pages(self):
             f'?term={self.QUOTED_SIGNLE_RESULT_TERM}&pageType=category'
         )
         self.assertTrue(len(json_to_dict(response)) == 1)
+
+
+class Redirects(TestCase):
+
+    fixtures = ['dump.json']
+
+    # will be resurrected at rf#140
+    @unittest.expectedFailure
+    def test_redirect_on_existing_page(self):
+        """DB based redirect from existing url should do, but should not avoid it."""
+        # take some existing `url_from`
+        url_from = '/catalog/categories/category-0/tags/6-v/'
+        response = self.client.get(url_from)
+        self.assertEqual(response.status_code, 200)
```

Comment:
It is an unnecessary assertion, we should test only redirect logic here

Reply:
@artemiy312 , it's specific of this test case.

`django.contrib.redirects` provide different behaviour if `url_from` returns 200 or 404

Reply:
@duker33 but we already test it in other tests, isn't it?

Reply:
@artemiy312 , it's new test, that fails.
Maybe we touch redirects in some way in different code. In products scope, for example.
But it's doesn't matter for this task.

Here we test mech for redirects concept itself